frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -28,18 +29,15 @@ export function ElectionStatusPage() {
       </NavBar>
       <header>
         <section>
-          <h1>Eerste zitting</h1>
+          <h1>{t("election_status.first_session")}</h1>
         </section>
       </header>
       {statuses.length > 0 && statuses.every((s) => s.status === "definitive") && (
         <Alert type="success">
-          <h2>Alle stembureaus zijn twee keer ingevoerd</h2>
-          <p>
-            De resultaten van alle stembureaus in jouw gemeente zijn correct ingevoerd. Je kunt de uitslag nu definitief
-            maken en het proces verbaal opmaken. Doe dit alleen als er vandaag niks meer herteld hoeft te worden.
-          </p>
+          <h2>{t("election_status.difinitive.title")}</h2>
+          <p>{t("election_status.difititive.message")}</p>
           <Button onClick={finishInput} size="md">
-            Invoerfase afronden
+            {t("election_status.difititive.finish_button")}
```

Comment:
If we're not translating to English for now, should we have English or Dutch keys? 

Finding good English keys is almost like translating already.. (triggered by this misspelling of "definitive")

Reply:
Would suggest using english, because this will make it a lot easier to spot missing translations. Unless the translation engine does something fancy to mark missing strings.